tools/makemanifest.py

```diff
@@ -115,7 +127,18 @@ def system(cmd):
         return -1, er.output
 
 def convert_path(path):
-    return path.replace('$(MPY)', TOP)
+    #Perform variable substituion. TODO: Other ENV vars?
```

Comment:
Linters can flag TODOs so the question is: is it really needed to have this here, or is this rather something which you wonder about yourself? Also: could be fixed by using argparse and having it parse arbitrary key-value pairs.